mod_reforged/hooks/ai/tactical/behaviors/ai_attack_throw_net.nut

```diff
@@ -4,6 +4,18 @@
 	q.onEvaluate = @(__original) function( _entity )
 	{
 		local ret = __original(_entity);
+		// It is a generator in vanilla, but we add the typeof check for maximum
+		// compatibility if another mod changed the original function to not be a generator.
```

Comment:
Are you sure, vanilla does not simply crash, when `onEvaluate` is no longer a generator? They call `resume` on its return value